app.py

```diff
@@ -311,60 +173,76 @@ def main():
     if current_pair == stable_coin_pair:
         for stable_coin in src.settings.DEBT_TOKENS[:-1]:
             debt_token = stable_coin
-            main_chart_data, collateral_token_price = process_liquidity(main_chart_data, collateral_token, debt_token)
+            main_chart_data, collateral_token_price = process_liquidity(
```

Comment:
You have removed the `process_liquidity` function.

Reply:
Sorry, it's bad fixing merge conflict, fixed.